main/inc/lib/usermanager.lib.php

```diff
@@ -2291,7 +2291,10 @@ public static function get_extra_user_data_by_value($field_variable, $field_valu
           }
 
           } */
-        $where = "field_variable='".Database::escape_string($field_variable)."' AND field_value='".Database::escape_string($field_value)."'";
+          
+        $field_value = Database::escape_string($field_value);
```

Comment:
You should use a PHP code instead of using FIND_IN_SET. I'm not sure if that function exists in other DB drivers.